osu.Game.Tests/Visual/Online/TestSceneFriendDisplay.cs

```diff
@@ -24,31 +27,50 @@ public class TestSceneFriendDisplay : OsuTestScene
 
         protected override bool UseOnlineAPI => true;
 
+        [Resolved]
```

Comment:
Should be marked as
```suggestion
        [Resolved(CanBeNull = true)]
```
(well, either that, or the null checks in tests should be removed).